File: Arrays/Easy/26/26.py
from typing import List

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        dup = 0

        for i in range(1, len(nums)):
            if nums[i] == nums[i - 1]:
                dup += 1
            else:
                nums[i - dup] = nums[i]

        return len(nums) - dup

File: Arrays/Easy/26/26.c
#include <stdio.h>

int removeDuplicates(int* nums, int numsSize){
    int dup = 0;
    for(int i = 1; i < numsSize; i++){
        if(nums[i] == nums[i - 1]){
            dup++;
        }else{
            nums[i - dup] = nums[i];
        }
    }
    return numsSize - dup;
}